src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/appsettings.json

```diff
@@ -31,6 +31,8 @@
     "RuntimeCookieName": "AltinnStudioRuntime",
     "SBLBaseAdress": "https://at22.altinn.cloud/",
     "RoleCacheTimeout": 5,
+    "MainUnitCacheTimeout": 5,
+    "KeyrolePartyIdsCacheTimeout": 5,
```

Comment:
A users list of key role parties are normally very stable, although changes from ER revoking DAGL etc might occur at any time. Being able to selectively invalidate cache entries across all pods would be required in order to cache this for a extended amount of time, which is obviously way out of scope for this change.

But, as with the main unit cache, this can still probably be increased to several minutes to better account for multiple requests needing authorization within a short time frame.